src/components/badge/index.tsx

```diff
@@ -28,7 +28,7 @@ const Badge = ({
 	const hasLabel = !!ariaLabel
 	const isIconOnly = hasIcon && !hasText
 	const isStatusBadge =
-		color == 'success' || color == 'warning' || color == 'error'
```

Comment:
This `error` colour was itself an `error`, it didn't work, in the adjacent stylesheets we use `critical` as the colour name, which matches [the HDS spec for Badge](https://helios.hashicorp.design/components/badge#color).

Reply:
Do we have any instances of using 'error' as the color prop?

Reply:
I looked but couldn't find anything!